src/Arcus.Templates.WebApi/appsettings.json

```diff
@@ -15,5 +15,6 @@
     "Audience": "<your-audience>"
   },
   //#endif
-  "MyConfigKey": "MyConfigValue"
+  "MyConfigKey": "MyConfigValue",
+  "APPINSIGHTS_INSTRUMENTATIONKEY": ""
```

Comment:
Will have to be moved within the `//#if (Serilog)` as the Application Insights is only used when the project is configured using Serilog.

Reply:
https://github.com/arcus-azure/arcus.templates/pull/613/commits/ff1f837ca6693a58cb7543b63f1572bdb807ec57